scripts/apache/get_fasta_sequence

```diff
@@ -28,27 +29,60 @@ sub get_fasta_sequence {
     my ($server) = @_;
 
     my $id = $server->require_argument('id');
+
+    my @raw_id_array = split (',', $id);
+    $id = "";
+    foreach my $current_id (@raw_id_array) {
+        if (index($current_id, '-') != -1) {
+          $current_id =~ s/\.\d+//;
+        }
+        $id = $id . ',' . $current_id;
+    }
+
     my @queries;
-    my $query = 'http://www.ebi.ac.uk/Tools/dbfetch/dbfetch?format=fasta&style=raw&db=refseqn&id='
-      . $id;
+    my $query = 'http://www.ebi.ac.uk/Tools/dbfetch/dbfetch?format=fasta&style=raw&db=uniprotkb&id=';
     push @queries, $query;
-    $query = 'http://www.ebi.ac.uk/Tools/dbfetch/dbfetch?db=ena_sequence&format=fasta&style=raw&Retrieve=Retrieve&id='
-      . $id;
+    my $query = 'http://www.ebi.ac.uk/Tools/dbfetch/dbfetch?format=fasta&style=raw&db=refseqn&id=';
```

Comment:
you are doing 'my' on a reused variable.

Reply:
fixed